src/proto/FunctionRpc.proto

```diff
@@ -144,10 +141,7 @@ message StatusResult {
   repeated RpcLog logs = 3;
 }
 
-// TODO: investigate grpc heartbeat - don't limit to grpc implemention
-
-// Message is empty by design - Will add more fields in future if needed
-message WorkerHeartbeat {}
+// TODO: investigate grpc heartbeat - don't limit to grpc implementation
```

Comment:
nit: Is this todo still needed?

Reply:
Hmmm not sure, this was added 4 years ago, maybe there is a different user story for it?

Reply:
didn't find anything to track this, might not be relevant 4 years later so will remove unless some says not to :)